packages/tempus/src/Tempus_StepperLeapfrog_decl.hpp

```diff
@@ -12,7 +12,11 @@
 #include "Tempus_config.hpp"
 #include "Tempus_StepperExplicit.hpp"
 #include "Tempus_StepperObserverComposite.hpp"
```

Comment:
You should be able to move #include "Tempus_StepperObserverComposite.hpp" inside TEMPUS_HIDE_DEPRECATED_CODE.